plugins/woocommerce/client/admin/client/launch-your-store/hub/sidebar/tasklist.tsx

```diff
@@ -66,9 +66,25 @@ export const getLysTasklist = async () => {
 	// This is because LYS and the Home screen share the same task list, but the completion logic is different.
 	tasklist[ 0 ].tasks.forEach( ( task: TaskType ) => {
 		if ( task.id === 'payments' ) {
-			task.isComplete =
-				task.additionalData?.wooPaymentsHasOnlineGatewaysEnabled ??
-				false;
+			let isComplete = false;
+
+			// Store has other online gateways enabled.
+			if (
+				task.additionalData?.wooPaymentsHasOnlineGatewaysEnabled &&
+				! task.additionalData?.wooPaymentsIsOnboarded
+			) {
+				isComplete = true;
+			}
```

Comment:
I think we need `else if` here because otherwise we risk having the second condition overwriting the first and the intent doesn't seem to be that - I know they are mutually exclusive due to `task.additionalData?.wooPaymentsIsOnboarded`, but it might not stay like that in the future.

Or just do a single `if` with OR conditions.